plugins/org.jkiss.dbeaver.ui.editors.data/src/org/jkiss/dbeaver/ui/controls/resultset/ResultSetUtils.java

```diff
@@ -177,17 +173,53 @@ static String formatRowCount(long rows) {
         return rows < 0 ? "0" : String.valueOf(rows);
     }
 
-    public enum OrderingMode {
+    public enum OrderingPolicy {
+        DEFAULT(ResultSetMessages.pref_page_database_resultsets_label_order_policy_default),
```

Comment:
Do we really need `DEFAULT` here? It's the same for all databases and doesn't change dynamically. I would leave only two options